docs/projeto/arquitetura.md

```diff
@@ -232,24 +235,48 @@ Depois de usar o modelo acima, foi percebido pontos de melhoria e outro modelo m
 }
 ```
 
+### 6.2 Dados das populações das cidades
+Os dados populacionais das cidades também são armazenados no MongoDB em que cada estado é uma *collection*. Exemplo de nome para uma collection de população: `populations_df` para o Distrito Federal.
+
+Os objetos ou documentos na *collection* estão seguindo esse modelo:
+```
+{
+    "_id": 1,
+    "capture_data": "04/08/2020",
+    "year": 2020,
+    "cities": [
+        {
+            "name": "Águas Claras",
+			"population": 50000
+        },
+        {
+            "name": "Taguatinga",
+			"population": 150000
+        }
+    ]
+}
+```
+
+
+
 
-### 6.2 Dados dos usuários
+### 6.3 Dados dos usuários
 
 Os dados do usuário estão sendo armazenados no PostgreSQL com a seguinte modelagem:
 
-#### 6.2.1 Diagrama Entidade-Relacionamento
+#### 6.3.1 Diagrama Entidade-Relacionamento
 Esse diagrama mostra quais e como são as entidades e os relacionamentos entre elas. 
 ![DE-R](../images/architecture/user-service-DER.jpg)
 
-#### 6.2.2 Diagrama Lógico
+#### 6.3.2 Diagrama Lógico
 O modelo lógico dá mais detalhes de como estão implementadas as tabelas no banco de dados.
 ![Lógico](../images/architecture/user-service-logic.jpg)
 
 Os campos do tipo *enum* tem os seguintes valores possíveis:
 
 * **detalhesAvaliacao (tabela avalia):** ("iluminação ruim", "pouca movimentação de pessoas", "poucas rondas policiais", "boa iluminação", "boa movimentação de pessoas", "rondas policiais frequentes")
 * **arma (tabela Ocorrencia)**: ("nenhuma", "de fogo", "branca")
-* **tipoOcorrencia (tabela Ocorrencia)**: ("Latrocínio", "Roubo a transeunte", "Roubo de Veículo", "Roubo de Residência" , "Estupro")
+* **tipoOcorrencia (tabela Ocorrencia)**: ("Latrocínio", "Roubo a transeunte", "Roubo de Veículo", "Roubo de Residência" , "Estupro", "Roubo a Transeunte", "Roubo de Veículo")
```

Comment:
The values "Roubo a Transeunte" and "Roubo de Veículo" are duplicated. Shoudn't it be "Furto" instead?

Reply:
You're right. The types have been fixed